system_metrics_collector/src/system_metrics_collector/linux_process_memory_measurement_node.cpp

```diff
@@ -91,3 +98,11 @@ uint64_t GetProcessUsedMemory(const std::string & statm_process_file_contents)
 }
 
 }   // namespace system_metrics_collector
+
+
+#include "rclcpp_components/register_node_macro.hpp"
```

Comment:
Include at the top of the file? Why here?

Reply:
The composition demos did this. Sure, I'll move it to the top of the file.

Reply:
Done.